packages/integrations/src/offline.ts

```diff
@@ -0,0 +1,101 @@
+import { captureEvent } from '@sentry/minimal';
+import { Event, EventProcessor, Hub, Integration } from '@sentry/types';
+import { getGlobalObject, logger, uuid4 } from '@sentry/utils';
+// @ts-ignore
+import localforage = require('localforage');
```

Comment:
Any reason we are not using import statement here?

Reply:
😄  There sure is... I'll see if I can link some open issues on localforage, but the gist is this: localforage does not export modules in a way compatible with this library. I attempted each of the suggested fixes (via the README and related issues), but I believe there is actually a bug in the package.

Reply:
Huh interesting. I will take a look, maybe it is a problem with our rollbar/bundling setup.

Reply:
It's definitely possible that I just missed something, but I tried quite a few suggestions. The issues I ran into can be reproduced by following the [suggestions in the localforage README for typescript](https://github.com/localForage/localForage#typescript).

Reply:
I'm going to mark this as resolved, because I ended up figuring out what was wrong.